app/views/users/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<p style="color: green"><%= notice %></p>
+
+<div class="d-flex flex-column" style="height:100vh;">
+  <div class="mb-auto d-flex align-items-center justify-content-center h-100">
+    <h1 class="text-center" >CountApp</h1>
+  </div>
+
+  <div class="d-flex flex-column mx-3">
+    <%= link_to "Log In", new_user_session_path, class: 'btn btn-primary p-3' %>
+    <%= link_to "sign up", new_user_registration_path, class: 'btn btn-light p-3' %>
+  </div>
+</div>
```

Comment:
- [ ] Your app in general is too big. Let me explain. When you get into the app, it renders perfectly but with the issue that it is too big and you cannot see the entire view on the screen. This is not a problem of screen size because I resized my window to try to fit the app but it simply does not fit in it. I think it would be great to reduce some sizes, especially if there are lots of blank spaces. I'll drop some images here so you can see my point of view:

![Captura de pantalla_2022-05-27_04-01-03](https://user-images.githubusercontent.com/59100477/170614753-adddefa8-a43d-48a5-8b7b-c312c4a5f073.png)
![Captura de pantalla_2022-05-27_04-01-30](https://user-images.githubusercontent.com/59100477/170614754-8547a6ff-7552-4bed-b1f1-a357c51236b6.png)
![Captura de pantalla_2022-05-27_04-02-26](https://user-images.githubusercontent.com/59100477/170614755-e178ff27-ee9d-47ba-8eb2-ea1689367c9f.png)
![Captura de pantalla_2022-05-27_04-03-10](https://user-images.githubusercontent.com/59100477/170614757-00adad77-f0b4-456f-82b0-e4a6e48fc291.png)